lib/KDateRange/__tests__/KDateRange.spec.js

```diff
@@ -120,16 +120,18 @@ describe('KDateCalendar component', () => {
     expect(wrapper.exists()).toBe(true);
   });
 
-  it('the month and year display on the left-side calendar view should contain the previous month', async () => {
+  it('the month and year display on the left-side calendar view should display the previous month of the lastAllowedDate', async () => {
     const previousMonthDisplay = getKDateCalendarElements(wrapper).previousMonthDisplay();
-    const previousMonth = new Date().setDate(0);
+    const lastAllowedDate = wrapper.props().lastAllowedDate;
+    const previousMonth = new Date(lastAllowedDate) - 1;
```

Comment:
I think a safest way of getting the previous month could be
```suggestion
    const previousMonth = new Date(lastAllowedDate);
    previousMonth.setMonth(lastAllowedDate.getMonth() - 1);
```

And it will not depend if the lastAllowedDate is the first day of the month

Reply:
Yes, good catch! Since the `lastAllowedDate` is manually set to November in this test case, we won't have to worry about January dates, but it would definitely be the safer approach.